vocab/types/VOCAB_QUDT-TYPES-v2.0.ttl

```diff
@@ -246,16 +246,13 @@ qudt:UTF8-CHAR
   dcterms:description "The QUDT 'Datatype' vocabulary is a specification of scalar and structured data types. Scalar data types are defined for all the commonly encountered forms of numbers, characters and booleans, with their representations in a number of vendor and industry specific technologies. Structured data types cover arrays, lists, trees and many other forms of composite data elements. Specification of coordinate systems is also covered." ;
   dcterms:modified "2020-04-08"^^xsd:date ;
   dcterms:subject "Datatypes" ;
-  vaem:hasGraphRole vaem:SchemaGraph ;
+  vaem:hasGraphRole vaem:VocabularyGraph ;
   vaem:isMetadataFor <http://qudt.org/2.0/vocab/datatype> ;
-  vaem:latestPublishedVersion "http://www.qudt.org/doc/2019/10/DOC_SCHEMA-QUDT-DATATYPES-v2.1.html"^^xsd:anyURI ;
   vaem:logo "http://www.linkedmodel.org/lib/lm/images/logos/qudt_logo-300x110.png"^^xsd:anyURI ;
   vaem:name "QUDT" ;
-  vaem:namespace "http://qudt.org/schema/qudt/" ;
+  vaem:namespace "http://qudt.org/vocab/type/" ;
   vaem:namespacePrefix "qudt.type" ;
   vaem:owner "QUDT.ORG" ;
-  vaem:previousPublishedVersion "http://www.qudt.org/doc/2017/DOC_SCHEMA-QUDT-DATATYPES-v2.0.html"^^xsd:anyURI ;
-  vaem:publishedDate "2017-05-10"^^xsd:date ;
   vaem:releaseDate "2020-04-08"^^xsd:date ;
   vaem:revision "2.0" ;
   vaem:title "QUDT Vocabulary for Datatypes - Version 2.0" ;
```

Comment:
1. Should change the turtleFileURL (which is really the file name) to ...vocab...
2. Do we want to "publish" the datatypes vocabulary on the web, or just wait for access via Explorer?
If we do want to publish, we'll need the vaem:latestPublishedVersion and vaem:previousPublishedVersion properties back (with vocab names for the html, of course).
3. Consider making this revision 2.1, just for consistency with the other vocabs.

Reply:
Changed 2.0 to 2.1. Deleted 'vaem:latestPublishedVersion' and 'vaem:previousPublishedVersion' statements.